client/environment.go

```diff
@@ -52,3 +52,29 @@ func (c *APIClient) UpdateEnvironment(req models.UpdateEnvironmentRequest) (stri
 
 	return resp.JobID, nil
 }
+
+func (c *APIClient) CreateLink(sourceID, destinationID string) error {
```

Comment:
This should either be moved out of this PR or the unit tests should be added

Reply:
agreed. ✅

Reply:
I assume we are going the unit test route?

Reply:
Yes. The API changes are in a separate PR #401 . Wanted to confirm whether we expect link/unlink should be returning jobs or not. ✅

Reply:
ok cool :+1: ...we should probably take that to slack